src/AsyncApiValidator.js

```diff
@@ -1,19 +1,18 @@
+'use strict'
+
 const {fail} = require('assert')
 const Ajv = require('ajv')
-const deref = require('json-schema-deref-sync')
 
 class AsyncApiValidator {
   /**
    * @param {any} schema
    * @param {any} asyncapiSchema
    */
   constructor(schema, asyncapiSchema) {
-    this._schema = schema || fail('schema is mandatory')
     this._asyncapiSchema = asyncapiSchema || fail('asyncapi is mandatory')
+    this._schema = schema || fail('schema is mandatory')
+    this._messages = this._schema.components.messages
```

Comment:
Can we always assume that `components.messages` will be there?

Reply:
No, there are some other edge cases also.
I need to work on this. 👍

Reply:
Ok, let's leave it for another PR.